### ISSUE TITLE
Add close tab settings

### ISSUE THREAD

**Issue by bbondy** [brave/browser-laptop#6553]
It would be nice to provide options for what to do when a tab is closed. Currently we will return to the parent tab first, and if no parent go to the next. 

Cross posting from community.brave.com:

```
Normally these are the possibilities when we close an active tab.

1.Activate the last active tab
The default setting returns focus to the most recently used tab
2.Activate the next tab
This setting returns focus to the next tab on the tab bar
3.Activate first tab 
But I don't see a preference setting like these in the "Tab Setting" section. This would be a nice addition.
```

Reference:
https://community.brave.com/t/tab-focus-when-an-active-tab-is-closed-if-there-are-multiple-tabs-open/540

**Comment by NejcZdovc** [comment]
@bbondy can I tackle this issue?

**Comment by luixxiul** [comment]
There are other options to be added: #2137

**Comment by NejcZdovc** [comment]
@luixxiul Do you think that they should be part of this PR? I can add them is separate PR's. Or are you talking about close tab options directly?

**Comment by luixxiul** [comment]
@NejcZdovc No I don't think so, I just related them for references ;-)

**Comment by luixxiul** [comment]
Oh, this PR implements the setting of *closing* tabs, while in #2137 the setting of *opening* new tabs is suggested.

If this function can be implemented in a similar way, I think you're the proper person to implement it :+1: if you don't mind.

**Comment by NejcZdovc** [comment]
Sure will implement after this closing tab PR.

**Comment by luixxiul** [comment]
#6573 was merged into `0.13.1-branch`.

**Comment by luixxiul** [comment]
It works perfectly! I like it :-D
